src/app/schedule-section.tsx

```diff
@@ -0,0 +1,93 @@
+import Image from "next/image"
+
+const schedule=[
```

Comment:
Formatting of this JSON is not consistent with the rest of the project.

I think I've not been the best with single vs double quotes, but you should have spaces around the equals sign, colons, etc.

Reply:
this is literally copy pasted from info.tsx

Reply:
not sure what you're referring to by `info.tsx` - doesn't exist in this repo

Reply:
done